x-pack/plugins/cases/public/common/navigation/deep_links.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { AppDeepLink } from '../../../../../../src/core/public';
+import { DEFAULT_BASE_PATH, getCreateCasePath, getCasesConfigurePath } from './paths';
+
+export const CasesDeepLinkId = {
+  cases: 'cases',
```

Comment:
@semd I was just wondering why casesDetail is not listed here.

Reply:
Yes, good question. This is because caseDetails has a dynamic path, the deepLinks need to be defined with static paths. 
In consequence, the way we navigate to case details is a little bit different than the other pages, we use the main `cases` deepLink and the `path` property, with the detailName parameter. Navigations:
https://github.com/elastic/kibana/pull/117582/files#diff-cfcdf6d32c2fb4f5fbe9a982c048819bb0dd82b30192c712c98e2cd0b484a4a9